media/js/firefox/firstrun/firstrun.js

```diff
@@ -35,7 +34,12 @@
             document.getElementById('sunrise').addEventListener('transitionend', function(event) {
                 if (event.propertyName === 'transform') {
                     window.setTimeout(function () {
-                        window.location.href = redirectDest;
+                        // Specially navigate to about:newtab and focus address bar
+                        if (redirectDest === 'about:newtab') {
+                            Mozilla.UITour.showNewTab();
```

Comment:
This results in `TypeError: Mozilla.UITour is undefined`. It looks like the UITour lib is not included in the [bundle for this page](https://github.com/mozilla/bedrock/blob/master/bedrock/settings/static_media.py#L1218-L1224).